### PR TITLE
add delete photo business logic on FE

### PR DESCRIPTION
**Pull request by rostyk1331**

### CODE REVIEW

File: client/src/store/profile/actions.ts [hunk 1]
```diff
@@ -29,3 +36,29 @@ export const getUser = (user: IUser): (dispatch: Dispatch) => Promise<void> =>
...
+
+export const deletePhoto = (): (dispatch: Dispatch) => Promise<void> =>
+    async (dispatch: Dispatch): Promise<void> => {
+        try {
+            dispatch(deletePhotoPending());
+            const res = await AuthAPI.delete('/profile/photo');
+            dispatch(deletePhotoSuccess(res.data.photoPath));
+            dispatch(showAlert(res.data.status, 'success'));
+        } catch (e) {
+            dispatch(deletePhotoError(e));
```
Comment (InnaKutsik) [inline client/src/store/profile/actions.ts]: no error message?

Comment (rostyk1331) [reply client/src/store/profile/actions.ts]: fixed.

File: client/src/components/PopUp/PopUp.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ interface IModalState {
     modal: boolean;
 }
 
-export default class PopUpModal extends React.Component<{}, IModalState> {
+export default class PopUpModal extends React.Component<any, IModalState> {
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: any?

File: client/src/components/PopUp/PopUp.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ interface IModalState {
...
     constructor(props: {}) {
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: ?

Comment (rostyk1331) [reply client/src/components/PopUp/PopUp.tsx]: Fixed.

File: client/src/components/PopUp/PopUp.tsx (hunk 2 above)
Comment (rostyk1331) [reply client/src/components/PopUp/PopUp.tsx]: Fixed.

**Pull request merged**